<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerInterface</class>
 <widget class="QDialog" name="PlayerInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PlayerWindow{
}</string>
  </property>
  <widget class="QGroupBox" name="PlayerArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>471</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#RetailerArea {
	background-color: #fdfbf2;
}</string>
   </property>
   <property name="title">
    <string>Player</string>
   </property>
   <widget class="QPushButton" name="orderBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>League Spartan Bold</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">
#orderBtn {
    color:#ffcc4e;
    background-color:#007bff;
    border-color:#007bff;
    font-family: 'League Spartan Bold', arial;
    font-weight: bold;
    border: 1px solid #f5f5f5; 
}

#orderBtn:hover { background-color: #66c011; color: white; border: 1px solid black;}

#orderBtn:disabled {
	background-color: #add8e6;
	color: black;
}</string>
    </property>
    <property name="text">
     <string>Place Order</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="orderIn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="totalCost_l">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total Cost:</string>
    </property>
   </widget>
   <widget class="QLabel" name="incomingShipment">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>+123</string>
    </property>
   </widget>
   <widget class="QLabel" name="outgoingShipment">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>-123</string>
    </property>
   </widget>
   <widget class="QLabel" name="weekCost_l">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Week's Cost:</string>
    </property>
   </widget>
   <widget class="QLabel" name="incoming_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Incoming Ship:</string>
    </property>
   </widget>
   <widget class="QLabel" name="outgoing_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Outgoing Ship:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="currentBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Current:</string>
    </property>
    <widget class="QLabel" name="inventory_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Inventory:</string>
     </property>
    </widget>
    <widget class="QLabel" name="backLog_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>BackLog:</string>
     </property>
    </widget>
    <widget class="QLabel" name="inventory">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>123</string>
     </property>
    </widget>
    <widget class="QLabel" name="backLog">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>123</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="afterBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>After:</string>
    </property>
    <widget class="QLabel" name="inventoryL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Inventory:</string>
     </property>
    </widget>
    <widget class="QLabel" name="backLog_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>BackLog:</string>
     </property>
    </widget>
    <widget class="QLabel" name="inventoryAfter">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>123</string>
     </property>
    </widget>
    <widget class="QLabel" name="backlogAfter">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>123</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="incomingOrder">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>-123</string>
    </property>
   </widget>
   <widget class="QLabel" name="outgoing_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Incoming Order:</string>
    </property>
   </widget>
   <widget class="QLabel" name="weekCost">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalCost">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalCost_l_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>310</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Week:</string>
    </property>
   </widget>
   <widget class="QLabel" name="week">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <zorder>currentBox</zorder>
   <zorder>orderBtn</zorder>
   <zorder>orderIn</zorder>
   <zorder>totalCost_l</zorder>
   <zorder>incomingShipment</zorder>
   <zorder>outgoingShipment</zorder>
   <zorder>weekCost_l</zorder>
   <zorder>incoming_2</zorder>
   <zorder>outgoing_2</zorder>
   <zorder>afterBox</zorder>
   <zorder>incomingOrder</zorder>
   <zorder>outgoing_4</zorder>
   <zorder>weekCost</zorder>
   <zorder>totalCost</zorder>
   <zorder>totalCost_l_2</zorder>
   <zorder>week</zorder>
  </widget>
  <widget class="QGroupBox" name="UpstreamGraph">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>0</y>
     <width>471</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#UpstreamGraph {
	background-color: #ededed;
}</string>
   </property>
   <property name="title">
    <string>From Upstream: Orders and Shipment Graph</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="ConsumerDemandGraph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>471</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#ConsumerDemandGraph {
	background-color: #ededed;
}</string>
   </property>
   <property name="title">
    <string>Consumer-Demand Graph</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="DownstreamGraph">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>471</width>
     <height>341</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#DownstreamGraph {
	background-color: #ededed;
}</string>
   </property>
   <property name="title">
    <string>To DownStream: Orders and Shipment Graph</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
